db.js

```diff
@@ -63,14 +66,16 @@ exports.init = function (sbot, dir, config) {
   }
 
   function get(id, cb) {
```

Comment:
is this the only API that needs to be guarded with `onIndexesReady`?

Reply:
It would be better in query, but that api is sync

Reply:
We could use a `deferred` to create a wait. Also, could we combine `onDrain` and `onIndexesReady` and have just one way of waiting? Does that make sense?

Reply:
Deferred would be a good idea. I think those are two different things. onIndexesReady is a boot thing whole the other is for when data is added later. Maybe a better name would be onIndexStateLoaded instead of onIndexesReady?